packages/aws-cdk-lib/region-info/build-tools/fact-tables.ts

```diff
@@ -4,25 +4,26 @@ export const AWS_CDK_METADATA = new Set([
   'ap-east-1',
   'ap-northeast-1',
   'ap-northeast-2',
-  // 'ap-northeast-3',
+  'ap-northeast-3',
   'ap-south-1',
-  // 'ap-south-2',
+  'ap-south-2',
   'ap-southeast-1',
   'ap-southeast-2',
+  'ap-southeast-3',
   'ap-southeast-4',
   'ca-central-1',
   'cn-north-1',
   'cn-northwest-1',
   'eu-central-1',
-  // 'eu-central-2',
+  'eu-central-2',
   'eu-north-1',
   'eu-south-1',
-  // 'eu-south-2',
+  'eu-south-2',
```

Comment:
Revert, we have not deployed the metadata resource in this region

Reply:
Fixed! Thanks @mrgrain